src/main/java/org/opentripplanner/standalone/CommandLineParameters.java

```diff
@@ -70,6 +69,15 @@
             description = "Pass the graph to the server in-memory after building it, and saving to disk.")
     public boolean preFlight;
 
+    @Parameter(names = { "--version", },
+            description = "Print the version, and then exit.")
+    public boolean version = false;
+
+    @Parameter(names = { "--version-fragment" },
+            description = "Print the a fragment of the version, and then exit. Use one of [`version`, `major`, `minor`, `incremental`, " +
+                    "`qualifier`, `commit` ] to specify which fragment")
+    public String versionFragment = DEFAULT_VERSION_FRAGMENT;
```

Comment:
Do we really need an option to select the fragment? If we just always output all version fragments, it's very easy to machine-parse and pull out the elements we need since each element is on a separate line and begins with the name of the fragment.